index.html

```diff
@@ -8216,6 +8217,107 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="suggestion">
+			<rdef>suggestion</rdef>
+			<div class="role-description">
+					<p>A single proposed change to content.</p>
+					<p>For example, in an editing system that supports multiple users, one user may suggest a change, and another user would be responsible
+					for accepting or rejecting the suggestion.</p>
+					<p>Authors MUST ensure that a <code>suggestion</code> contains either one <rref>insertion</rref> child or one <rref>deletion</rref> child or ensure that it contains two children where one is an <rref>insertion</rref> and the other is a <rref>deletion</rref>. Authors MUST ensure a <code>suggestion</code> does not contain any other children.</p>
+					<p>Authors MAY use <pref>aria-details</pref> or <pref>aria-description</pref> to associate the <code>suggestion</code> with related information such as comments, authoring info, and time stamps.</p>
+					<pre class="example highlight">
+						&lt;p&gt;
+						  The best pet is a
+						  &lt;span role="suggestion"&gt;
+						    &lt;span role="deletion"&gt;cat&lt;/span&gt;
+						    &lt;span role="insertion"&gt;dog&lt;/span&gt;
+						  &lt;/span&gt;
+						&lt;/p&gt;
+					</pre>
+					<p>When a suggestion is accepted, authors SHOULD remove the <code>suggestion</code> role, indicating that the proposed revision has been made.
+						In addition, child <rref>insertion</rref> and <rref>deletion</rref> elements can be retained or transformed into the final accepted content.
```

Comment:
The words "in addition" kind of imply this is an extension of the SHOULD statement. I don't think it is. I think it is just informative authoring guidance. I recommend the following to improve clarity:

```suggestion
						After the <code>suggestion</code> role is removed, child <rref>insertion</rref> and <rref>deletion</rref> elements can either be retained to document the revision or replaced with the revised content.
```